app/controllers/facility_facility_accounts_controller.rb

```diff
@@ -26,7 +26,7 @@ def new
 
   # POST /facilities/:facility_id/facility_accounts(.:format)
   def create
-    @facility_account = current_facility.facility_accounts.new(params[:facility_account])
+    @facility_account = current_facility.facility_accounts.new(facility_facility_account_params)
```

Comment:
Can we use `load_and_authorize_resource through: :current_facility` and then get rid of this line and the `find` in update?

Reply:
I can't get that to work--I think it's confused about the class name, despite it being passed explicitly in the `load_and_authorize_resource`. It looks for `facility_facility_account` on current_facility.